controllers/velero.go

```diff
@@ -427,23 +428,16 @@ func removeDuplicateValues(slice []string) []string {
 }
 
 func (r *DPAReconciler) customizeVeleroDeployment(dpa *oadpv1alpha1.DataProtectionApplication, veleroDeployment *appsv1.Deployment) error {
-	veleroDeployment.Labels = r.getAppLabels(dpa)
-	veleroDeployment.Spec.Selector = &metav1.LabelSelector{
-		MatchLabels: map[string]string{
-			"app.kubernetes.io/name":       common.Velero,
-			"app.kubernetes.io/instance":   dpa.Name,
-			"app.kubernetes.io/managed-by": common.OADPOperator,
-			"app.kubernetes.io/component":  Server,
-			oadpv1alpha1.OadpOperatorLabel: "True",
-		},
+	//append dpa labels
+	for k, v := range r.getDpaAppLabels(dpa) {
+		if veleroDeployment.Labels[k] == "" { //saves component: velero labels
```

Comment:
We set if it's not set already 
preserves label from https://github.com/openshift/oadp-operator/pull/638/files#r849903351